ground_station/src/main.cpp

```diff
@@ -43,6 +41,9 @@ void setup() {
 
 void loop() {
   static bool ini{false};
+  static uint16_t link1LastTs{0};
```

Comment:
we have 16-bit timestamps?

Reply:
Yeah, the telemetry sends down 16 bit timestamps